src/components/organisms/Certifications/index.tsx

```diff
@@ -75,17 +77,48 @@ const Certifications = () => {
     setActiveIndex(key);
   };
 
-  const getCertificateData = (item: CertificateData) => {
-    const data = {
-      id: activeIndex,
-      image: item.Image,
-      name: item.name,
+  const getBase64 = (file: any, cb: any) => {
+    const reader = new FileReader();
+    reader.readAsDataURL(file);
+    reader.onload = function () {
+      cb(reader.result);
     };
+  };
+
+  const getCertificateInfo = async () => {
+    await axios
+      .get(`http://localhost:8000/certificates?portfolio_id=${1}`)
```

Comment:
Call all axios functions in a separate services file and import those functions here. And store the base url in a variable so that you can reuse